integration-tests/src/test/java/uk/gov/di/authentication/api/AuthenticationCallbackHandlerIntegrationTest.java

```diff
@@ -309,7 +310,7 @@ void shouldLogoutAndRedirectToBlockedPageWhenAccountStatusIsBlocked()
                         constructQueryStringParameters());
 
         assertRedirectToBlockedPage(response);
-        assertOrchSessionIsUpdatedWithUserInfoClaims();
```

Comment:
I assume these were removed because when the Orch session gets deleted these fail? Even though the Orch session does get updated with userinfo claims before it's deleted

Reply:
Yea exactly: the tests fail because we can't get the orch session at this point, so `orchSessionExtension.getSession(SESSION_ID)` throws